FrontEnd/main/src/components/cookieacception/CookieMod.js

```diff
@@ -0,0 +1,54 @@
+import { Cookies } from 'react-cookie';
+import { useCookies } from 'react-cookie';
+import styles from './CookieMod.module.css';
+import cookiesimage from './cookieimg/homecookies.png';
+
+const CookieMod = ({ active, setActive }) => {
+  const [cookies, setCookie, removeCookie] = useCookies();
+  console.log(cookies.first);
+  const accessCookie = () => {
+    let d = new Date();
+    d.setTime(d.getTime() + 30 * 24 * 60 * 60 * 1000);
+    setCookie('first', true, { expires: d, sameSite: 'lax' });
+    setActive(false);
+  };
+
+  const denyCookie = () => {
+    let d = new Date();
+    d.setTime(d.getTime() + 30 * 24 * 60 * 60 * 1000);
+    setCookie('first', false, { expires: d, sameSite: 'lax' });
+    setActive(false);
+  };
+  return cookies.first ? (
+    ''
```

Comment:
Usually we return `null`

Reply:
Can you explain in more detail what's the problem?

Reply:
I have changed '' to null. Did you mean this detail?

Reply:
@BelousSofiya , just found interesting [article](https://medium.com/@davidkelley87/stop-using-return-null-in-react-a2ebf08fc9cd). Looks like its better to return `false`.

Reply:
Thank you for usefull resources!